### DISCUSSION TITLE
Icon for `cover` entities

### DISCUSSION THREAD

**Discussion by sfuqua** [cgiesche/streamdeck-homeassistant#289] [Feature Requests]
I've got a toggle button for Lutron shades in my office so that I can open/close them. Would be awesome if this plugin supported the Home Assistant iconography for covers/shades, but in a pinch, any sort of visual indicator for open/shut would be amazing.

**Comment by sfuqua** [comment]
Suddenly I am seeing an icon for this entity - not sure what changed. It is a garage door icon which I can see in the default display config. Not sure if there is a way for the package to disambiguate between e.g. a garage door and a window shade.

**Comment by cgiesche** [comment]
I was working on a complete rewrite of the "button rendering" that has been released yesterday. So your icon was a neat side effect.
If your door and window shade are entities of the same type, my plugin will show the same icon for both.

You could change the icon in Home-Assistant and configure the plugin to prefer the HA-Icon, but then there will be the same icon for both open and closed state :/
